source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,79 @@
+.. _ruby-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: installation, setup, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure you have the following dependencies
+      installed in your development environment:
+      
+      - `Ruby <https://www.ruby-lang.org/en/downloads/>`__ version 2.7
+        or later
+      - `bson <https://rubygems.org/gems/bson>`__ version 4.14.1 or later, up to
```

Comment:
`bson` gem is installed automatically when installing the Ruby driver, there is no need to install it separately.

Reply:
Ah okay! I'll take it out